store/templates/store/store_index_page.html

```diff
@@ -6,40 +6,67 @@
     <h1>{{ page.title }}</h1>
     {{ page.intro | richtext }}
 
-    {% if products.count %}
-        {% for product in products.all %}
-            <div class="card mb-2">
-                <div class="card-body">
-                    <h2>
-                        <a href="{% pageurl product %}">
-                            {{ product }}
-                        </a>
-                    </h2>
-                    <div class="container">
-                        <div class="row">
-                            <div class="col-2">
-                                {% image product.image max-150x150 class="float-left me-2" %}
-                            </div>
-                            <div class="col">
-                                {{ product.description | richtext | truncatewords_html:30 }}
-                            </div>
+<!-- Group and Sort Books by is_feautered -->
+    {% regroup products|dictsort:"title" by is_featured as grouped_products %}
```

Comment:
> Note that {% regroup %} does not order its input! 
> The easiest solution to this gotcha is to make sure in your view code that the data is ordered according to how you want to display it.
> https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#regroup

This is a subtlety of the `regroup` tag, but the data need to be sorted by the grouping element.

```suggestion
    {% regroup products|dictsort:"is_featured" by is_featured as grouped_products %}
```

Also, the items can be sorted in the database query to leverage any database optimizations.